src/atlas/unzipAtlas.m

```diff
@@ -22,6 +22,12 @@ function unzipAtlas(atlas)
 
     end
 
+  elseif strcmp(atlas, 'visfAtlas')
+
+    file = fullfile(atlasDir, 'visfAtlas/space-MNI_atlas-visfAtlas_dseg.nii.gz');
```

Comment:
```suggestion
    file = fullfile(atlasDir, 'visfAtlas', 'space-MNI_atlas-visfAtlas_dseg.nii.gz');
```

Reply:
should be fine one most OS because fullfile is usually smart enough to fix those things but just in case